src/components/staff-payment/StaffPayment.vue

```diff
@@ -168,15 +162,24 @@ export default class StaffPayment extends Vue {
   })
   readonly staffPaymentData!: StaffPaymentIF
 
-  /** Radio group model property. */
-  private paymentOption = StaffPaymentOptions.NONE
-
   /** FAS form model property. */
   private fasFormValid = false
 
+  /** If FAS form inputs are disabled. */
+  private get fasFormInputDisabled (): boolean {
+    return this.staffPaymentData.option === StaffPaymentOptions.BCOL ||
+      this.staffPaymentData.option === StaffPaymentOptions.NO_FEE
+  }
+
   /** BCOL form model property. */
   private bcolFormValid = false
```

Comment:
I recommend keeping local variables together, and computeds (getters) together (and near the top of the class).